.changeset/early-balloons-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': minor
```

Comment:
Would be a `patch` bump since it's not breaking and we're on 0.x

Reply:
Done!